mteb/models/voyage_models.py

```diff
@@ -156,6 +156,13 @@ def _batched_encode(
     max_tokens=16000,
     embed_dim=1024,
     open_source=False,
+    n_parameters=None,
+    memory_usage=None,
+    license=None,
+    reference="https://blog.voyageai.com/2024/05/05/voyage-large-2-instruct-instruction-tuned-and-rank-1-on-mteb/",
+    similarity_fn_name="cosine",
+    framework=["Sentence Transformers", "PyTorch"],
```

Comment:
Voyage models are API only. Maybe change framework to API?

Reply:
fixed!